di-ipv-cimit-stub/lambdas/get-contra-indicators/src/main/java/uk/gov/di/ipv/core/getcontraindicators/domain/ContraIndicatorItem.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.di.ipv.core.getcontraindicators.domain;
+
+import lombok.Data;
+
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+
+@Data
+public class ContraIndicatorItem implements Comparable<ContraIndicatorItem> {
```

Comment:
ContraIndicatorItem in the stub won't need to be Comparable

Reply:
done